cli/command/context/create.go

```diff
@@ -0,0 +1,105 @@
+package context
+
+import (
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/context/docker"
+	"github.com/docker/cli/cli/context/kubernetes"
+	"github.com/docker/cli/cli/context/store"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func newCreateCommand(dockerCli command.Cli) *cobra.Command {
+	opts := &createOptions{}
+	cmd := &cobra.Command{
+		Use:   "create [OPTIONS] NAME",
+		Short: "Create a context",
+		Args:  cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.name = args[0]
+			return runCreate(dockerCli, opts)
+		},
+	}
+	flags := cmd.Flags()
+	flags.StringVar(&opts.description, "description", "", "Description of the context")
+	flags.StringVar(
+		&opts.defaultStackOrchestrator,
+		"default-stack-orchestrator", "",
+		"Default orchestrator for stack operations to use with this context (swarm|kubernetes|all)")
+	flags.BoolVar(&opts.use, "use", false, "Use this context by default")
```

Comment:
What about `--use-as-default` ? Just `--use` does not feel really self explanatory...